test/models/baseTransaction.ts

```diff
@@ -42,9 +42,11 @@ describe("Transaction Verification", function () {
       ],
       Signers: [
         {
-          Account: "r....",
```

Comment:
Why were the changes in this file necessary?

Reply:
The test case was incorrectly formatted - It didn't include the "Signers: { ... }" wrapper which rippled transactions have to have.